src/main/java/com/matchus/global/error/GlobalExceptionHandler.java

```diff
@@ -28,4 +29,15 @@
 			ErrorCode.INVALID_INPUT_VALUE, e.getBindingResult());
 		return new ResponseEntity<>(errorResponse, HttpStatus.BAD_REQUEST);
 	}
+
+	@ExceptionHandler(
+		{SportsNotFoundException.class,
+			UserNotFoundException.class
+		}
+	)
```

Comment:
```suggestion
	@ExceptionHandler(
		{
		        SportsNotFoundException.class,
			UserNotFoundException.class
		}
	)
```

이렇게 하면 보기 더 좋을 것 같습니다!